src/inflight_activation_store/store_tests.rs

```diff
@@ -111,7 +132,12 @@ async fn test_store_duplicate_id_between_batches() {
 #[tokio::test]
 async fn test_get_pending_activation() {
     let url = generate_temp_filename();
-    let store = InflightActivationStore::new(&url).await.unwrap();
+    let store = InflightActivationStore::new(
+        &url,
+        InflightActivationStoreConfig::from_config(&create_integration_config()),
+    )
+    .await
+    .unwrap();
```

Comment:
Should we have a test_utils method that builds store instances now that the setup is a bit more involved?

Reply:
Added util function for this 👍